packages/jellyfish-api-core/__tests__/category/vault/estimateVault.test.ts

```diff
@@ -0,0 +1,165 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import BigNumber from 'bignumber.js'
+import { TestingGroup } from '@defichain/jellyfish-testing'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+import { TokenInfo } from '@defichain/jellyfish-api-core/dist/category/token'
+
+describe('Vault estimateVault', () => {
+  const tGroup = TestingGroup.create(1, i => new MasterNodeRegTestContainer())
+
+  async function setup (tGroup: TestingGroup): Promise<void> {
+    // token setup
+    const collateralAddress = await tGroup.get(0).container.getNewAddress()
+    await tGroup.get(0).token.dfi({ address: collateralAddress, amount: 30000 })
+    await tGroup.get(0).generate(1)
+    // Setup collateral token
+    await tGroup.get(0).token.create({ symbol: 'BTC', collateralAddress })
+    await tGroup.get(0).generate(1)
+    await tGroup.get(0).token.mint({ symbol: 'BTC', amount: 20000 })
+    await tGroup.get(0).generate(1)
+    // Setup non collateral token
+    await tGroup.get(0).token.create({ symbol: 'DOGE', collateralAddress })
+    await tGroup.get(0).generate(1)
+    await tGroup.get(0).token.mint({ symbol: 'DOGE', amount: 50000 })
+    await tGroup.get(0).generate(1)
+
+    // oracle setup
+    const addr = await tGroup.get(0).generateAddress()
+    const priceFeeds = [
+      { token: 'DFI', currency: 'USD' },
+      { token: 'BTC', currency: 'USD' },
+      { token: 'DOGE', currency: 'USD' },
```

Comment:
You may remove `DOGE` from priceFeeds as it is not used as collateral tokens or loan tokens.
Yeah, you use it in your test unit but the test unit written is a negative test. So, no harm.